indico/queries/usermetrics.py

```diff
@@ -0,0 +1,113 @@
+from typing import List
+
+from indico.client.request import (
+    GraphQLRequest,
+    PagedRequest,
+)
+from indico.types import BaseType
+from indico.types.user_metrics import UserSummary, UserSnapshot
+
+
+class _PagedUserSnapshots(BaseType):
+    """
+    Class to hold paged snapshot data to make parsing easier.
+    """
+    results: List[UserSnapshot]
+
+
+class _UserMetricsFilters(BaseType):
+    """Filters for fetching user metric data."""
+    date: str
+    user_id: str
+    user_email: str
+
+    def __init__(self, **kwargs):
+        self.date = kwargs.get('date', None).strftime('%Y-%m-%d') if kwargs.get('date', None) is not None else ""
```

Comment:
```suggestion
        self.date = kwargs['date'].strftime('%Y-%m-%d') if kwargs.get('date') else None
```
and can the default be None or does it have to be `""`